lib/registries/addon/components/comment-card/template.hbs

```diff
@@ -55,60 +65,82 @@
                             {{t (concat 'registries.overview.comments.abuse_' category)}}
                         </form.select>
                         <form.textarea
-                            @valuePath="message"
+                            @valuePath='message'
                             @label={{t 'registries.overview.comments.report_placeholder'}}
                         />
-                        <div local-class="formButtons">
-                            <button type="cancel" local-class="CommentReport__cancel" class="btn btn-default btn-sm" {{action this.cancelReport}}>
+                        <div local-class='formButtons'>
+                            <OsfButton
+                                local-class='CommentReport__cancel'
+                                @type='default'
+                                @size='sm'
+                                @onClick={{action this.cancelReport}}
+                            >
                                 {{t 'registries.overview.comments.cancel'}}
-                            </button>
-                            <button type="submit" local-class="CommentReport__save" class="btn btn-default btn-sm" disabled={{form.disabled}}>
+                            </OsfButton>
+                            <OsfButton
+                                data-analytics-name='Report comment'
+                                local-class='CommentReport__save'
+                                type='submit'
+                                @type='default'
+                                @size='sm'
+                                @disabled={{form.disabled}}
+                            >
                                 {{t 'registries.overview.comments.report'}}
-                            </button>
+                            </OsfButton>
                         </div>
                     </ValidatedModelForm>
                 {{/if}}
             </div>
             {{#if (not this.reporting)}}
-                <div local-class={{concat "CommentControls " (or (if this.hasReplies '' 'noReplies') (if this.canReport '' 'cantReport'))}}>
+                <div local-class={{concat 'CommentControls ' (or (if this.hasReplies '' 'noReplies') (if this.canReport '' 'cantReport'))}}>
```

Comment:
Instead of adding all this logic and using `noReplies` and `cantReport` just to change `justify-content`, leave the divs whether or not they have buttons in them:
```hbs
<div local-class='CommentControls'>
  <div local-class='CommentControls__report'>
    {{#if this.canReport}}
      <OsfButton ...>
    {{/if}}
  </div>
  <div local-class='CommentControls__repliesCount'>
    {{#if this.hasReplies}}
      <OsfButton ...>
    {{/if}}
  </div>
</div>
```